cmd/cli/operations/update_job_test.go

```diff
@@ -11,39 +11,70 @@ import (
 )
 
 /*
- * filterRunningJobs
+ * filterRunningJobsByName
  */
 func TestFilterRunningJobsShouldReturnAnEmptySliceIfNoJobsWereProvided(t *testing.T) {
 	operator := RealOperator{}
 
-	res := operator.filterRunningJobs([]flink.Job{})
+	res := operator.filterRunningJobsByName([]flink.Job{}, "test")
 
 	assert.Len(t, res, 0)
 }
 
 func TestFilterRunningJobsShouldReturnAnEmptySliceIfNoJobsAreRunning(t *testing.T) {
 	operator := RealOperator{}
 
-	res := operator.filterRunningJobs([]flink.Job{
-		flink.Job{
-			Status: "STOPPED",
+	res := operator.filterRunningJobsByName(
+		[]flink.Job{
+			flink.Job{
+				Status: "STOPPED",
+			},
 		},
-	})
-
+		"test",
+	)
 	assert.Len(t, res, 0)
 }
 
 func TestFilterRunningJobsShouldReturnTheRunningJobs(t *testing.T) {
 	operator := RealOperator{}
 
-	res := operator.filterRunningJobs([]flink.Job{
-		flink.Job{
-			Status: "STOPPED",
+	res := operator.filterRunningJobsByName(
+		[]flink.Job{
+			flink.Job{
+				Status: "STOPPED",
+				Name:   "test",
+			},
+			flink.Job{
+				Status: "RUNNING",
+				Name:   "test",
+			},
 		},
-		flink.Job{
-			Status: "RUNNING",
+		"test",
+	)
+
+	assert.Len(t, res, 1)
+}
+
+func TestFilterRunningJobsShouldReturnTheRunningJobsMatchingTheJobBaseName(t *testing.T) {
+	operator := RealOperator{}
+
+	res := operator.filterRunningJobsByName(
+		[]flink.Job{
+			flink.Job{
+				Status: "STOPPED",
+				Name:   "jobA",
```

Comment:
Since we're switching to checking the prefix of the `job.Name`, we should have `jobA V1.0` and `jobA V1.1` here to verify the exact behaviour

Reply:
Done